agents/financial-advisor/deployment/test_deployment.py

```
@@ -0,0 +1,105 @@
+    print(f"Created session for user ID: {FLAGS.user_id}")
+    print("Type 'quit' to exit.")
+    while True:
+        user_input = input("Input: ")
+        if user_input == "quit":
+            break
+
+        for event in agent.stream_query(
+            user_id=FLAGS.user_id, session_id=session["id"], message=user_input
+        ):
+            if "content" in event:
+                if "parts" in event["content"]:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)